src/cli/requirements.txt

```diff
@@ -1,15 +1,15 @@
-msal~=1.4.3
```

Comment:
What changed here?

Reply:
Git was recognizing it as a binary file somehow. In build script I am defining to make it write as an ascii file.